### DISCUSSION TITLE
Question about tur-continuous

### DISCUSSION THREAD

**Discussion by T-Dynamos** [termux-user-repository/tur#338] [General]
Can I use it for building libreoffice? As libreoffice's docs say:
> Now you are ready for building using make. Beware that full (all components, non-incremental) build may take up to 24 hours and takes up about 20 GB of drive space. You will also most likely need at least 8 GBs of RAM, otherwise the machine might fall into swap and appear to freeze up.

**Comment by licy183** [comment]
Yes, you can try to compile it. But IMO you should try to compile it on your computer first because Github Action is much slower.
